lec11-modelling.Rmd

```diff
@@ -0,0 +1,527 @@
+However, they are limited in that the real-world often doesn't perfectly fit
+these models. The real world is messy and noisy. We can't always trust our data
+as there are inherent biases and errors in it. Measuring it, collecting it,
+recording it, and inputting it are some of the possible sources of error and
+bias.  We use statistics and probability to determine whether the way we
+understand and conceptualize the world (as models) matches reality (even with
+the error and bias).
+
+A reason we use statistical methods in R compared to writing up the formulas and
+equations ourselves is that we can focus on answering questions without worrying
+about whether we are doing the math or computation wrong. This is of course
+dependent on the type of research questions you may be interested in (e.g. for
+more theoretical questions, doing the math and computation yourself is probably
+a goal!) and on the type of data you are using/collecting. There is a lot of
+complexity that has already been taken care of in the available R packages and
+functions. For example, the function `glm` that we will use in this lesson takes
+a maximum likelihood-based approach to estimation, which is a complex method for
+determining fit (like the least squares method of fitting you did in the
+previous lesson). That way, you can answer your research questions and not worry
+too much about the exact math involved and instead worry about the specifics of
+your field (e.g. Are you measuring the right thing? Are you collecting the right
+data? Are you asking the right questions? Is there an ecological or biological
+aspect you are missing in your analysis?)
+
+## Generalized linear models
+
+Generalized linear models (or GLM) is a family of statistical modelling
+techniques, that can also be used in place of several other techniques, e.g.
+correlation or analysis of variance (ANOVA), linear regression, logistic
+regression. GLM is a powerful technique that you can use on a wide range of data
+and research questions, and is the foundation for understanding how more
+advanced techniques work. This is the reason I will be covering GLM in depth.
+(For those who will go into graduate students or into research in general, you
+will likely use mixed effects models. If you can understand and grasp GLM, it
+will be substantially easier for you to understand mixed effects modelling and
+other techniques.)
+
+A version of GLM that uses continuous $y$ values is called linear regression,
+which I'm going to focus on. The formula for linear regression (or GLM in
+general) is:
+
+$$ Y = \alpha + X\beta + \varepsilon $$
+
+Or, a simplified, alphabetized version is:
+
+$$ y = a + Xb + e $$
+
+Where $a$ is the intercept, $X$ is the variable, $b$ is the slope/coefficient,
+and $e$ is the error term. In the case of multiple linear regression (more than
+one $x$), it is expanded to:
+
+$$ y = a + x_1b_1 + x_2b_2 +...+ x_nb_n + e $$
```

Comment:
Why small `x` here, and capital `X` when there is only one `b`?

Reply:
There is a reason (X is a matrix of all the variables, while x is a single variable), but it isn't something that is important. I'll change it to small x.